packages/fx-core/src/plugins/resource/aad/errors.ts

```diff
@@ -15,8 +15,8 @@ export interface AadError {
 
 export const GetAppError: AadError = {
   name: "AadGetAppError",
-  message: (objectId: string) =>
-    `Failed to get app info with Object ID: ${objectId}. ` +
+  message: () =>
```

Comment:
message has to be a function?

Reply:
If message is a function, please follow function naming convention such as `getMessage()`.
Or, make it a property accessors:
```
export 
const GetAppError = {
  name: "AadGetAppError",
  message: ()=> {
      return "Failed to get app info with current Object Id in env.default.json. Please make sure object id is valid, " + `or delete 'objectId' under ${Plugins.pluginNameComplex} in env.default.json and try again.`;
  }
}
```